assets/src/containers/AssignmentPlanning.js

```diff
@@ -66,19 +66,9 @@ function AssignmentPlanning(props) {
   }
 
   const AssignmentTable = plan => (
-    <Table
+    <TableAssignment
       tableHead={['Week', 'Due', 'Title', 'Percent of final grade']}
-      tableData={generateAssignmentTable(plan)
-        .map(row => {
-          const { percentOfFinalGrade, graded } = row.pop()
-          row.push(<HorizontalBar
```

Comment:
Took the `HorizontalBar` component from the container and added to the `TableAssignment` component. Felt passing the Data is simple(line in 71 here) without nesting the component inside the data. Thoughts??